resources/user.go

```diff
@@ -196,12 +201,27 @@ func (obj *UserRes) CheckApply(apply bool) (checkOK bool, err error) {
 		if obj.GID != nil {
 			args = append(args, "-g", fmt.Sprintf("%d", *obj.GID))
 		}
+		if obj.Group != nil {
+			args = append(args, "-g", *obj.Group)
+		}
+		if obj.Groups != nil {
+			var groups string
+			for i, group := range obj.Groups {
```

Comment:
use the strings.Join method instead
since this gets joined with comma, we'll definitely have to validate this list as well in Validate